applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -498,16 +495,12 @@ static void stream_enabled_cb(struct bt_bap_stream *stream)
 
 static void stream_stop_cb(struct bt_bap_stream *stream, uint8_t reason)
 {
-	int ret;
+	LOG_DBG("Stream %p stopped. Reason: %d", stream, reason);
```

Comment:
I would suggest we use LOG_INF in here, since we use LOG_INF in stream_start_cb(), and that really help us to check if ISO setup properly.

Reply:
We need to have a quick discussion here, if these types of messages in the le_audio files should be INF or DBG. My viewpoint is that streamctrl should be INF, and le_audio should be DBG